sdk/python/foundation-models/system/finetune/image-classification/multilabel-classification/hftransformers-fridgeobjects-multilabel-classification.ipynb

```diff
@@ -194,16 +194,16 @@
    "source": [
     "### 3. Pick a foundation model to fine tune\n",
     "\n",
-    "We will use the `microsoft-beit-base-patch16-224-pt22k-ft22k` model in this notebook. If you need to fine tune a model that is available on HuggingFace, but not available in `azureml-staging` system registry, you can either register the model and use the registered model or use the `model_name` parameter to instruct the components to pull the model directly from HuggingFace.\n",
+    "We will use the `microsoft-beit-base-patch16-224-pt22k-ft22k` model in this notebook. If you need to fine tune a model that is available on HuggingFace, but not available in `azureml` system registry, you can either register the model and use the registered model or use the `model_name` parameter to instruct the components to pull the model directly from HuggingFace.\n",
     "\n",
     "Currently following models are supported:\n",
     "\n",
     "| Model Name | Source |\n",
     "| ------ | ---------- |\n",
-    "| [microsoft-beit-base-patch16-224-pt22k-ft22k](https://ml.azure.com/registries/azureml-staging/models/microsoft-beit-base-patch16-224-pt22k-ft22k/version/3?tid=72f988bf-86f1-41af-91ab-2d7cd011db47#overview) | azureml-staging registry |\n",
-    "| [microsoft-swinv2-base-patch4-window12-192-22k](https://ml.azure.com/registries/azureml-staging/models/microsoft-swinv2-base-patch4-window12-192-22k/version/3?tid=72f988bf-86f1-41af-91ab-2d7cd011db47#overview) | azureml-staging registry |\n",
-    "| [facebook-deit-base-patch16-224](https://ml.azure.com/registries/azureml-staging/models/facebook-deit-base-patch16-224/version/3?tid=72f988bf-86f1-41af-91ab-2d7cd011db47#overview) | azureml-staging registry |\n",
-    "| [google-vit-base-patch16-224](https://ml.azure.com/registries/azureml-staging/models/google-vit-base-patch16-224/version/3?tid=72f988bf-86f1-41af-91ab-2d7cd011db47#overview) | azureml-staging registry |\n",
+    "| [microsoft-beit-base-patch16-224-pt22k-ft22k](https://ml.azure.com/registries/azureml/models/microsoft-beit-base-patch16-224-pt22k-ft22k/version/3?tid=72f988bf-86f1-41af-91ab-2d7cd011db47#overview) | azureml registry |\n",
```

Comment:
Double checking...Would version 3 work on "azureml"?

Reply:
If we are releasing for the first time, it would be version "1", right?

Reply:
Also, should we remove tenant id from the query string? ("tid=72f988bf-86f1-41af-91ab-2d7cd011db47") The reason is, this is Microsoft tenant id, so it would work for us, but for our customers it would run into errors. If I remove the tenent id, it automatically appends my tenant id in the url.

![image](https://github.com/Azure/azureml-examples/assets/107195344/2774827d-159b-4b12-969e-07cc06200348)

Reply:
No, it would be the latest version.

Reply:
tenant id removed, version updated. 
Thanks for catching this, Nilesh.